<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_novaVenda</class>
 <widget class="QDialog" name="fm_novaVenda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txt_qtde">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_idProduto">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_listaProdutos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>700</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantidade</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CÃ³digo do produto</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_totalVenda">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor total R$ 0,00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_excluir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>110</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remover</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_editar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>110</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_venda">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>110</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Finalizar Venda</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pesquisar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>110</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pesquisar produto</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
